yoti-sdk-spring-boot-example/src/main/resources/templates/dynamic-share.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html SYSTEM "http://www.thymeleaf.org/dtd/xhtml1-strict-thymeleaf-spring4-4.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org" class="yoti-html">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <title>Yoti client example</title>
```

Comment:
Not sure if we have this in all the SDKs, but I prefer having a different title to the index page
```suggestion
  <title>Dynamic Share Example</title>
```